File: BuilderDesignPattern/CarManufacture_Example/Components/GPSNavigator.java
package BuilderDesignPattern.CarManufacture_Example.Components;

public class GPSNavigator {
	private String route;

	public GPSNavigator() {
		this.route = "4290 Albany Dr, San Jose, CA, 95129";
	}

	public GPSNavigator(String manulRoute) {
		this.route = manulRoute;
	}

	public String getRoute() {
		return route;
	}

	public void setRoute(String route) {
		this.route = route;
	}

	@Override
	public String toString() {
		return this.getRoute();
	}
}

File: BuilderDesignPattern/CarManufacture_Example_Python/Components/GPSNavigator.py
class GPSNavigator:
	_route: str

	def __init__(self):
		self._route = "4290 Albany Drive, San Jose, CA, 95129"

	def getRoute(self) -> str:
		return self._route

	def setRoute(self, destinationAddress: str):
		self._route = destinationAddress

	def __repr__(self) -> str:
		return self.getRoute()

	def __str__(self) -> str:
		return f"Route: {self._route}"